### PR TITLE
Changes to IAM footer

### PR DESCRIPTION
**Pull request by raja651**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:
Improve footer design and content as mentioned in [issue](https://github.com/Automattic/woocommerce.com/issues/21574)

Closes https://github.com/Automattic/woocommerce.com/issues/21574 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch
2. Go to /plugins/woocommerce and run pnpm --filter='@woocommerce/plugin-woocommerce' build
3. Visit the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions).
4. Now view the screen with less products one could be business services or search for random text like `sadasd` on search
Screenshot 2024-09-12 at 3 56 03 PM

5. Confirm footer now contained in a box similar to [design your own](https://href.li/?https://prnt.sc/6idFn5AuP3Ov) banner in Themes
6. Confirm the `woo marketplace` content on footer is now removed
7.  Also confirm `Support the ecosystem` content is added to the footer as the 4th element.
8. Test on different screen sizes and confirm nothing is broken on footer.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
Update footer design, add one more element to footer content and remove `woo-marketplace` copy at footer bottom.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx [hunk 1]
```diff
@@ -71,10 +72,14 @@ function FooterContent(): JSX.Element {
 						'woocommerce'
 					) }
 				/>
-			</div>
-			<div className="woocommerce-marketplace__footer-logo">
```
Comment (bor0) [inline plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx:75]: Do we also want to remove it from the scss file?

File: plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx [hunk 2]
```diff
@@ -37,6 +36,8 @@ const paymentTitle = createInterpolateElement(
 	}
 );
 
+const ecosystemTitle = __( 'Support the ecosystem', 'woocommerce' );
```
Comment (bor0) [inline plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx]: Do we need this constant? Why not inline like we do with description?

File: plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx [hunk 3]
```diff
@@ -71,10 +72,14 @@ function FooterContent(): JSX.Element {
...
-			<div className="woocommerce-marketplace__footer-logo">
-				<img src={ WooIcon } alt="Woo Logo" aria-hidden="true" />
-				<span>{ __( 'Woo Marketplace', 'woocommerce' ) }</span>
+				<IconWithText
+					icon={ people }
+					title={ ecosystemTitle }
+					description={ __(
+						'Our team and partners are continuously improving your extensions, themes, and WooCommerce experience.',
+						'woocommerce'
+					) }
```
Comment (bor0) [inline plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx:79]: I also thought of [this](https://github.com/Automattic/woocommerce.com/issues/21574#issuecomment-2344020224) - nice of you to mention it! Let's see if we need to add any links here.

**Review by bor0 — APPROVED** [review]
Changes look good and test well. Tested on playground both desktop and mobile views. Before merging let's check the links thing first.

File: plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx (hunk 2 above)
Comment (raja651) [reply plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx]: fixed it in [7870cd0](https://github.com/woocommerce/woocommerce/pull/51315/commits/7870cd0c86d5f35341f626d7d5c3d17fd2b7bf91)

**Pull request merged**